modules/cnf-image-based-upgrade-prep-oadp.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+// * edge_computing/image-based-upgrade/cnf-preparing-for-image-based-upgrade.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cnf-image-based-upgrade-creating-backup-oadp-resources_{context}"]
+= Creating OADP ConfigMap objects for the image-based upgrade with {lcao}
+
+Create your OADP resources that are used to back up and restore your resources during the upgrade.
+
+.Prerequisites
+
+* Generate a seed image from a compatible seed cluster.
+* Create backup and restore resources.
+* Create a separate partition on the target cluster for the container images that is shared between stateroots. For more information about, see _Additional resources_.
+* Deploy a version of {lcao} that is compatible with the version used with the seed image.
+* Install the OADP Operator, the `DataProtectionApplication` CR and its secret on the target cluster.
+* Create an S3-compatible storage solution and a ready-to-use bucket with proper credentials configured. For more information, see _Additional resources_.
+
+.Procedure
+
+. Create the OADP `Backup` and `Restore` CRs for platform artifacts in the same namespace where the OADP Operator is installed, which is `openshift-adp`.
+
+.. If the target cluster is managed by {rh-rhacm}, add the following YAML file for backing up and restoring {rh-rhacm} artifacts:
++
+--
+.PlatformBackupRestore.yaml for {rh-rhacm}
+include::snippets/ibu-PlatformBackupRestore.adoc[]
+--
+
+.. If you created persistent volumes on your cluster through {lvms}, add the following YAML file for {lvms} artifacts:
++
+.PlatformBackupRestoreLvms.yaml for {lvms}
+include::snippets/ibu-PlatformBackupRestoreLvms.adoc[]
+
+. (Optional) If you need to restore applications over the upgrade, create the OADP `Backup` and `Restore` CRs for your application in the `openshift-adp` namespace.
+
+.. Create the OADP CRs for cluster-scoped application artifacts in the `openshift-adp` namespace.
++
+.Example OADP CRs for cluster-scoped application artifacts for LSO and {LVMS}
+include::snippets/ibu-ApplicationClusterScopedBackupRestore.adoc[]
+
+.. Create the OADP CRs for your namespace-scoped application artifacts.
++
+--
+.Example OADP CRs namespace-scoped application artifacts when LSO is used
+include::snippets/ibu-ApplicationBackupRestoreLso.adoc[]
+
+.Example OADP CRs namespace-scoped application artifacts when {lvms} is used
+include::snippets/ibu-ApplicationBackupRestoreLvms.adoc[]
+
+[IMPORTANT]
+====
+The same version of the applications must function on both the current and the target release of {product-title}.
+====
+--
+
+. Generate a `ConfigMap` object for your OADP CRs.
+
+.. Create the `ConfigMap` object:
```

Comment:
Since there is only one substep and it seems to be creating the object, I would remove this and just have the step be "Create a `ConfigMap` object for your OADP CRs:"